backend/corpora/common/entities/dataset.py

```diff
@@ -224,10 +224,16 @@ def get_most_recent_artifact(self, filetype=DatasetArtifactFileType.CXG):
         )
         return DatasetAsset(artifact[0]) if artifact else None
 
-    def publish_new(self):
-        self.update(collection_visibility=CollectionVisibility.PUBLIC, published=True, commit=False)
+    def publish_new(self, now: datetime):
```

Comment:
Since this now returns something non obvious, I think it's worth adding some inline documentation to this function